### PR TITLE
DTSRD-1466-1.Bringing down pods rd-perftest

### PR DESCRIPTION
**Pull request by SabinaHMCTS**
### Jira link (if applicable)
DTSRD-1466

### Checklist

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

File: apps/rd/rd-user-profile-api/perftest.yaml [hunk 1]
```diff
@@ -6,7 +6,7 @@ spec:
   releaseName: rd-user-profile-api
   values:
     java:
-      replicas: 2
+      replicas: 0
```
[the remark this answers is not in the document]
Comment (SabinaHMCTS) [reply apps/rd/rd-user-profile-api/perftest.yaml:9]: i am tryign to bring down the pods in order to migrate data to V15 postgres

[the remark this answers is not in the document]
Comment (SabinaHMCTS) [reply apps/rd/rd-user-profile-api/perftest.yaml:9]: will you be please able to give me your approval for this PR so i can go ahead with the migration

[the remark this answers is not in the document]
Comment (pksaxena01) [reply apps/rd/rd-user-profile-api/perftest.yaml:9]: Thanks both Jon & Sabina for clarifications.

**Review by pksaxena01 — APPROVED** [review]
Approved

**Pull request merged**